test/test_grid_file.py

```diff
@@ -213,12 +226,14 @@ def test_grid_out_default_opts(self):
 
         gout = GridOut(self.db.fs, 5)
         with self.assertRaises(NoFile):
+            gout.open()
```

Comment:
There's a lot of added open() calls in this file. Is that intended or should they be changed to `if not _IS_SYNC: open()`

Reply:
They should all be only for the async tests, fixed.